tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -3622,8 +3622,9 @@ lldbg_emit_local_variable(LL_DebugInfo *db, SPTR sptr, int findex,
       flags = 0;
     }
     var_mdnode = lldbg_create_local_variable_mdnode(
-        db, DW_TAG_auto_variable, blk_info->mdnode, symname, file_mdnode, 0, 0,
-        type_mdnode, flags, fwd, sptr);
+        db, DW_TAG_auto_variable, blk_info->mdnode, PASSBYVALG(sptr)
+       ? SYMNAME(MIDNUMG(sptr)) : symname, file_mdnode, 0, 0,
```

Comment:
Avoid breaking the ternary expression like this, or please format the parameter list as clang-format would.